aio/gulp/backend.js

```diff
@@ -30,8 +30,6 @@ gulp.task('backend', gulp.series((doneFn) => {
   goCommand(
       [
         'build',
-        // Install dependencies to speed up subsequent compilations.
-        '-i',
```

Comment:
Any reason to remove this?

Reply:
Yes, there is warning in the build logs that this flag is deprecated. From https://golang.org/cmd/go/: "The -i flag installs the packages that are dependencies of the target. The -i flag is deprecated. Compiled packages are cached automatically.".